openshift/e2e-common.sh

```diff
@@ -86,88 +86,15 @@ function install_strimzi(){
 
 function install_serverless(){
   header "Installing Serverless Operator"
-  git clone --branch release-1.6 https://github.com/openshift-knative/serverless-operator.git /tmp/serverless-operator
+  git clone --branch master https://github.com/openshift-knative/serverless-operator.git /tmp/serverless-operator
+  # cp openshift/olm/serverless-operator.v1.8.0.clusterserviceversion.yaml /tmp/serverless-operator/olm-catalog/serverless-operator/1.8.0/serverless-operator.v1.8.0.clusterserviceversion.yaml
```

Comment:
this is a marker, that this we we can add overrides, to e.g. `knative-nightly` if needed - that's all